s3file/static/s3file/js/s3file.js

```diff
@@ -1,4 +1,7 @@
 'use strict';
+var fileProgress = {};
```

Comment:
I would prefer to store those values in `window`, since the script could be loaded multiple times.

Reply:
latest commit removes the need for these.